pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -705,6 +725,28 @@
   dance_choose_activity: "Choose your Dance Party"
   dance_afe_disclaimer: "The artists whose music are used in this tutorial are not sponsoring or endorsing Amazon as part of licensing use of their music to Code.org."
 
+  minecraft_hero_heading: "Minecraft Hour of Code Tutorials"
+  minecraft_activities_desc: "All tutorials are available in many languages and compatible with modern browsers and tablets. Don't have internet? [Download the offline version of Minecraft Adventurer!](%{offline_url})"
+  minecraft_cdo_activity_title_aquatic: "Minecraft Voyage Aquatic"
+  minecraft_cdo_activity_desc_aquatic: "Build and explore with Minecraft! Use your creativity and problem solving skills to explore and build underwater worlds with code."
+  minecraft_cdo_activity_title_journey: "Minecraft Hero's Journey"
+  minecraft_cdo_activity_desc_journey: "Build and explore with Minecraft! Players write code that instructs the Agent to execute their commands and overcome in-game obstacles."
+  minecraft_cdo_activity_title_adventurer: "Minecraft Adventurer"
+  minecraft_cdo_activity_desc_adventurer: "Learn the basics of computer science by programming Alex or Steve to move through a simulated piece of a Minecraft world."
+  minecraft_cdo_activity_title_designer: "Minecraft Designer"
+  minecraft_cdo_activity_desc_designer: "You make the rules in this activity, where you can learn basic computer science skills to create your own Minecraft game to share with others!"
+  minecraft_education_activities_heading: "Available in Minecraft Education"
+  minecraft_education_activity_title_2023: "Minecraft Generation AI"
+  minecraft_education_activity_desc_2023: "Venture through time to create your AI-powered inventions! The new Minecraft Hour of Code tutorial is now available in Minecraft Education for Windows, Mac, Chromebook, iPad and Mobile. Get started with free educator resources, trainings, and videos."
+  minecraft_education_activity_title_2022: "Minecraft Escape Estate"
+  minecraft_education_activity_desc_2022: "Escape the mysterious mansion by dawn and earn a million emeralds!"
+  minecraft_education_activity_title_2021: "Minecraft TimeCraft"
+  minecraft_education_activity_desc_2021: "Travel back in time to save the future in this free coding lesson in Minecraft Education."
+  minecraft_education_activity_title_2020: "Minecraft AI for Earth"
+  minecraft_education_activity_desc_2020: "Program the Minecraft Agent to collect data about forest fires. Learn coding basics and explore a real-world example of artificial intelligence."
+  minecraft_resources_desc_01: "Utilize this on demand [free course](%{course_url}) or register your entire class to attend a live Hour of Code workshop with a guest teacher. Offered at multiple times during CSEdWeek, %{campaign_date}."
+  minecraft_resources_desc_02: "Inspire STEM passion in K-12 learners with 200hr of computer science curriculum, from block to python, supported by professional development."
```

Comment:
Should this say "200 hours" or is "200hr" okay?

Reply:
Oh yes, that sounds much better. Updated!